jplag.frontend-utils/src/main/java/de/jplag/TokenPrinter.java

```diff
@@ -152,4 +195,16 @@ private static List<String> linesFromFile(File file) {
         }
         return Collections.emptyList();
     }
+
+    /**
+     * Returns the number of digits (including a minus) of the given number.
+     */
+    private static int digitCount(int number) {
+        if (number == 0) {
+            return 1;
+        }
+        int minusLength = number < 0 ? 1 : 0;
+        // The 'log10' variant is supposedly faster than the 'toString' variant.
+        return (int) Math.log10(Math.abs(number)) + minusLength + 1;
```

Comment:
Careful with premature optimization! As this is a testing tool, performance probably does not matter that much.

Reply:
I agree! To be clear, I created this method myself, so I just tried to optimize my very own code, not code that existed before. I thought I would leave this comment to explain why I chose this approach.